tests/memberclient.py

```diff
@@ -53,33 +47,59 @@ def test_add_member(network, args):
     except infra.member.NoRecoveryShareFound as e:
         assert e.response.error == "Only active members are given recovery shares"
 
-    new_member.ack(primary)  # Activate new member
+    new_member.ack(primary)
 
-    new_share = already_active_member.get_and_decrypt_recovery_share(primary)
-    assert (
-        saved_share != new_share
-    ), "New shares should be issued when a new member is activated"
+    return network
 
 
 @reqs.description("Retire an existing member")
-def test_retire_member(network, args):
+@reqs.sufficient_member_count()
+def test_retire_member(network, args, member_to_retire=None):
+    primary, _ = network.find_primary()
+
+    if member_to_retire is None:
+        member_to_retire = network.consortium.get_any_active_member()
+    network.consortium.retire_member(primary, member_to_retire)
+
+    return network
+
+
+@reqs.description("Issue new recovery shares (without re-key)")
+def test_update_recovery_shares(network, args):
+    primary, _ = network.find_primary()
+    network.consortium.update_recovery_shares(primary)
+    return network
+
+
+def assert_recovery_shares_update(func, network, args, **kwargs):
```

Comment:
Wrapping the "extended" part (i.e. checking that the recovery shares actually change) of the add/retire member and set recovery threshold tests so that they are only called from `memberclient.py`. When the suite is run, the "extended" part is skipped for conciseness/speed.